src/assets/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM amazonlinux:2
+FROM amazonlinux:2023
+RUN dnf --setopt=install_weak_deps=False -y install nginx
```

Comment:
It prevents the Recommended/Suggests dependencies (i.e. the weak ones) from being installed. I've added a comment in the `Dockerfile`(s) for this. Since this project is an example, I've added it everywhere, so that people are more likely to see it.